ibis-server/tests/routers/v2/connector/test_snowflake.py

```diff
@@ -261,14 +261,24 @@ def test_validate_rule_column_is_valid_without_one_parameter(manifest_str):
         assert response.status_code == 422
         assert response.text == "Missing required parameter: `modelName`"
 
-    @pytest.mark.skip(reason="Not implemented")
     def test_metadata_list_tables():
-        pass
+        response = client.post(
+            url=f"{base_url}/metadata/tables",
+            json={"connectionInfo": connection_info},
+        )
+        assert response.status_code == 200
```

Comment:
Please add more expected to like `test_trino`.

Reply:
I’ve updated the test. Could you please take a look when you have time and let me know if I’ve done it correctly? Thanks @grieve54706!